src/deepsparse/v2/operators/operator.py

```diff
@@ -97,8 +107,19 @@ def __call__(
                 inference_state=inference_state,
                 **kwargs,
             )
+
         if self.has_output_schema():
-            return self.output_schema(**run_output)
+            rtn = self.output_schema(**run_output)
+            if middleware is not None:
```

Comment:
we should also run this in the other case where there's no schema, using `run_output` as the `outputs`